@xen-orchestra/lite/src/components/NoData.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="noData">
+    <img alt="No data" class="img" src="../assets/undraw_bug_fixing.svg" />
```

Comment:
```suggestion
    <img alt="No data" src="../assets/undraw_bug_fixing.svg" />
```
You don't need to add a `class`. You can refere to the `img` element.

Reply:
Even is scoped styles, it's still recommended to add a class.

> Scoped styles do not eliminate the need for classes. Due to the way browsers render various CSS selectors, `p { color: red }` will be many times slower when scoped (i.e. when combined with an attribute selector). If you use classes or ids instead, such as in `.example { color: red }`, then you virtually eliminate that performance hit.

Source: https://vue-loader.vuejs.org/guide/scoped-css.html#also-keep-in-mind